### PR TITLE
Python assist

### PR DESCRIPTION
**Pull request by inflatus**
Want to merge the stale branch. The python assist branch will be depreciated.

### CODE REVIEW

File: checkip_duo.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import requests
 
-data = {'format': 'json'}
+data = {'format':'json'}
```
Comment (itsthejoker) [inline checkip_duo.py]: image

**Comment by itsthejoker** [comment]
`Deprecated`, which was something that took me forever to get the hang of. To `depreciate` something is to financially devalue it.

**Comment by inflatus** [comment]
Trying to go too fast and not paying attention. I meant to say deprecated.

File: ip_information.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # need to revist this to pull information by other means
 # script relies on user(s) OS installs
+# nmap, whois and host need to be installed
```
[the remark this answers is not in the document]
Comment (inflatus) [reply ip_information.py]: Have a requirements.txt file in the Master branch now. Guess that brings up a question. Should I start to separate the scripts into folders and use a requirements.txt file for each? Am thinking that may be the best way.

File: checkip_duo.py (hunk 1 above)
Comment (inflatus) [reply checkip_duo.py]: Sure I totally messed this up. It does pull the info but puts it in curly braces and single quotes.

File: ip_information.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (itsthejoker) [reply ip_information.py]: This is a great opportunity to do that. The easiest way to generate them is to run `pip freeze > requirements.txt`, which will generate a list of all installed packages and dump them into the file. Nice and easy, and that way you know you're not contaminating projects with possibly conflicting libraries.

**Pull request merged**